src/Dapper.AOT.Analyzers/Dapper.AOT.Analyzers.csproj

```diff
@@ -16,9 +16,11 @@
 
     <ItemGroup>
         <Compile Remove="AotGridReader.cs" />
-        <Compile Remove="InterceptsLocationAttribute.cs" />
+        <Compile Remove="InGeneration\DapperHelpers.cs" />
```

Comment:
maybe wildcard the folder for compile/embeddedresource?

Reply:
sure, made a wildcard here